### ISSUE TITLE
bug: keyboard accessibiilty.  Cannot reach bottom tabs.

### ISSUE THREAD

**Issue by slclark** [ionic-team/ionic-framework#28107]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

v6.x

### Current Behavior

When starting a new tabs template and running the app in the browser, I attempt to test keyboard accessibility by navigating with the keyboard's TAB button.  When doing so, the bottom tabs in the app are never highlighted. Instead, the keyboard navigation will highlight the in page link, then go back to the top and navigate the browser's buttons.  The bottom tabs are never highlighted but are instead skipped entirely.  This is bad for accessibility because if Ionic is used as a webapp, keyboard-only users can never navigate beyond the first tab.

### Expected Behavior

The bottom tabs should be highlighted after the in page links have finished being highlighted.  The bottom navigation tabs should never be skipped entirely.

### Steps to Reproduce

1. Generate a new ionic tabs starter template.  ionic start test tabs 
2. Run your app in the browser.  ionic serve
3. Attempt to navigate using only the keyboard.

### Ionic Info

[WARN] Error loading @ionic/angular package.json: Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './package' is
       not defined by "exports" in
       /Users/sclark/Documents/Development/git/mobile/test/test/node_modules/@ionic/angular/package.json
[WARN] Error loading @capacitor/ios package.json: Error: Cannot find module '@capacitor/ios/package'
       
       Require stack:
       - /Users/sclark/.nvm/versions/node/v19.3.0/lib/node_modules/@ionic/cli/lib/project/index.js
       - /Users/sclark/.nvm/versions/node/v19.3.0/lib/node_modules/@ionic/cli/lib/index.js
       - /Users/sclark/.nvm/versions/node/v19.3.0/lib/node_modules/@ionic/cli/index.js
       - /Users/sclark/.nvm/versions/node/v19.3.0/lib/node_modules/@ionic/cli/bin/ionic
[WARN] Error loading @capacitor/android package.json: Error: Cannot find module '@capacitor/android/package'
       
       Require stack:
       - /Users/sclark/.nvm/versions/node/v19.3.0/lib/node_modules/@ionic/cli/lib/project/index.js
       - /Users/sclark/.nvm/versions/node/v19.3.0/lib/node_modules/@ionic/cli/lib/index.js
       - /Users/sclark/.nvm/versions/node/v19.3.0/lib/node_modules/@ionic/cli/index.js
       - /Users/sclark/.nvm/versions/node/v19.3.0/lib/node_modules/@ionic/cli/bin/ionic

Ionic:

   Ionic CLI                     : 6.20.6 (/Users/sclark/.nvm/versions/node/v19.3.0/lib/node_modules/@ionic/cli)
   Ionic Framework               : not installed
   @angular-devkit/build-angular : 16.2.1
   @angular-devkit/schematics    : 16.2.1
   @angular/cli                  : 16.2.1
   @ionic/angular-toolkit        : 9.0.0

Capacitor:

   Capacitor CLI      : 5.3.0
   @capacitor/android : not installed
   @capacitor/core    : 5.3.0
   @capacitor/ios     : not installed

Utility:

   cordova-res : not installed globally
   native-run  : 1.7.2

System:

   NodeJS : v19.3.0 (/Users/sclark/.nvm/versions/node/v19.3.0/bin/node)
   npm    : 9.2.0
   OS     : macOS Monterey
